lib/ui/window.dart

```diff
@@ -508,9 +508,9 @@ class Locale {
 
   String _rawToString(String separator) {
     final StringBuffer out = StringBuffer(languageCode);
-    if (scriptCode != null)
+    if (scriptCode != null && scriptCode != '')
```

Comment:
isNotEmpty?

Reply:
There are other lines of code in this file that also compare to `''`...